From 0000000000000000000000000000000000000000 Mon Sep 17 00:00:00 2001
From: quh4gko8 <88831734+quh4gko8@users.noreply.github.com>
Date: Tue, 13 May 2025 14:41:30 +0000
Subject: [PATCH] Increment version for search engines data

Commands used for incrementing version:

```bash
    version_line=$(echo $(grep kCurrentDataVersion definitions/prepopulated_engines.json))

    version=$(echo $version_line | sed -r "s/.*\"kCurrentDataVersion\": //g")

    incremented_version=$(($version + 1)); incremented_version_line=$(echo $version_line | sed -r "s/$version/$incremented_version/g")

    sed -ri "s/$version_line/$incremented_version_line/g" definitions/prepopulated_engines.json

```
---
 definitions/prepopulated_engines.json | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/definitions/prepopulated_engines.json b/definitions/prepopulated_engines.json
index 80873cb..9c3e9a1 100644
--- a/definitions/prepopulated_engines.json
+++ b/definitions/prepopulated_engines.json
@@ -28,7 +28,7 @@
     // existing data should get a new version. Otherwise, existing data may
     // continue to be used and updates made here will not always appear.
     // Also then run tools/search_engine_choice/generate_search_engine_icons.py.
-    "kCurrentDataVersion": 209
+    "kCurrentDataVersion": 210
   },
 
   // The following engines are included in country lists and are added to the
